examples/dreamcast/raylib/tetris/src/tetris.cpp

```diff
@@ -0,0 +1,88 @@
+#include <raylib.h>
+#include "game/game.h"
+#include "constants/constants.h"
+#include "colors/colors.h"
+#include <iostream>
+
+#include <kos/init.h>
+#include <kos/dbgio.h>
+#include <dc/sound/stream.h>
+#include <dc/maple.h>
+#include <dc/maple/controller.h>
+#include <wav/sndwav.h>
+
+double lastUpdateTime = 0;
+
+static bool EventTriggered(double interval){
+    double currentTime = GetTime();
+    if(currentTime - lastUpdateTime >= interval){
+        lastUpdateTime = currentTime;
+        return true;
+    }
+    return false;
+}
+
+int main(){
```

Comment:
```suggestion
int main(int argc, char* argv[]){
```
1) Lets teach them right.
2) DID YOU KNOW. The previous declaration will give a warning with LTO enabled, since KOS forward declares main in `init.c` to have this signature, not `(void)` argument list.